tests/top-langs.test.js

```diff
@@ -36,6 +36,10 @@ const data_langs = {
             },
           },
         ],
+        pageInfo: {
```

Comment:
Maybe let's add tests to check whether the function successfully fetches two pages and whether it stops when the `hasNextPage` parameter becomes `false`?

Reply:
I added a test for fetching two pages

Reply:
@MatteoPierro Looks good to me. I attached the `hacktoberfest-accepted` label so you should see it show up in your Hacktoberfest profile. We will probably merge this later by introducing an environment variable allowing us to disable this in the public instance (see #2159).